test/e2e/index.test.ts

```diff
@@ -587,6 +587,24 @@ async function initializeEnvironment(
         expect(groupHeaderWidth).to.equal(groupHeaderColumnsTotalWidth);
         expect(subGroupHeaderWidth).to.equal(subGroupHeaderColumnsTotalWidth);
       });
+
+      it('should not crash when updating columns immediately after scrolling', async () => {
+        await renderFixture('DataGrid/DynamicVirtualizationRange');
+
+        await page.mouse.move(200, 200);
+        await page.mouse.wheel(0, 1000);
+
+        let thrownError: Error | null = null;
+        context.once('weberror', (webError: WebError) => {
+          thrownError = webError.error();
+          console.error(thrownError);
+        });
+
+        await page.click('"Update columns"');
+
+        await sleep(200);
```

Comment:
is there a way to react on some callback to check if the error happened instead of waiting for some fixed time?

Reply:
It feels like an end to end test is a bit too heavy (slow) for this bug. It seems that to reproduce, it's only a matter of scrolling down to create a frozen context and update the columns to break part of the frozen context.

Reply:
Yeah, karma is faster.
Here are the results for running the same test 100 times:

- Karma:
	```
	Chrome Headless 125.0.6422.26 (Mac OS 10.15.7): Executed 100 of 6802 SUCCESS (5.168 secs / 4.887 secs)
	```
	
- Playwright:
	```
	Running on: chromium, version: 125.0.6422.26.
	  100 passing (15s)
	```

So Karma is ~3x faster.
I've added a test in Karma instead of the e2e test 👍🏻